lib/open_project/passwords.rb

```diff
@@ -101,21 +101,31 @@ def self.min_length
 
       # Returns a text describing the active password complexity rules,
       # the minimum number of rules to adhere to and the total number of rules.
-      def self.rules_description
+      # if receive true as parameter, the rules will be displayed as html list
```

Comment:
Why?

Beside the point of having no need in the first place to built this into a `lib` class, why do you merge the cases? this is non-speaking nonsense as the user of this function could never even guess, what the `true` parameter will do for them.